presto-docs/src/main/sphinx/release/release-0.270.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.270
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix error classification when an invalid timezone is passed as a parameter to :func:`from_unixtime`.
+* Improve performance of ``DISTINCT LIMIT N`` queries for N <= 10000. This can be enabled with the session property ``hash_based_distinct_limit_enabled``
+  or the configuration property ``hash-based-distinct-limit-enabled`` and the limit can be adjusted by using the session property ``hash_based_distinct_limit_threshold``
+  or the configuration property ``hash-based-distinct-limit-threshold``.
+* Add :func:`last_day_of_month` UDF to return the last day of the month.
+* Add dynamic filtering support for right join.
+* Add support for any expression for dynamic filtering probe side.
```

Comment:
I had a hard time parsing this one, but the alternatives I've come up with aren't much better. Maybe someone else has an idea.  If not this is fine.